app/utils/shared/renderMultilineText.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+
+const renderMultilineText = (text: string, useLineBreakElement = true) => {
+    const newLineCharacter = '\n';
```

Comment:
As `newLineCharacter` is being used strictly as a const, it may be moved out of the scope of the function execution 🚀 

```suggestion
import React from 'react';

const newLineCharacter = '\n';

const renderMultilineText = (text: string, useLineBreakElement = true) => {
```